handlers/homu_status/__init__.py

```diff
@@ -36,9 +49,27 @@ def check_failure_log(api, bors_comment):
     stdio = api.get_page_content(failed_url)
     failures = iter(re.findall(r'.*Tests with unexpected results:\n(.*)\n</span><span',
                                stdio, re.DOTALL)).next()
-    if failures:
-        comment_body = '\n'.join(map(lambda line: ' ' * 4 + line, failures.split('\n')))
-        api.post_comment(comment_body)
+    if not failures:
+        return
+
+    comment_body = '\n'.join(map(lambda line: ' ' * 4 + line, failures.split('\n')))
+    test_names = parse_log(failures)
+    if not test_names:
+        return
+
+    known_fails = []
+    intermittents = api.get_issue_titles(['I-intermittent'])
+    for num, title in intermittents:
+        for test_name in test_names:
+            if title.endswith(test_name):
+                known_fails.append(num)
+
+    if known_fails:
+        suffix = 'Possible intermittent(s) we can try for: ' + \
+                 ', '.join(map(lambda n: '#' + str(n), known_fails))
```

Comment:
I'd like to see a bulleted list of each issue number followed by the issue title.